src/components/Layout/SearchModal.tsx

```
@@ -1,6 +1,7 @@
 import React, { useState, useEffect, useRef } from "react";
 import { PageView, type CompanySearchItem } from "../../types";
 import { Search, X, ChevronRight, TrendingUp, Loader2 } from "lucide-react";
+import { searchCompanies, type CompanySearchItem } from "../../api/company";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'CompanySearchItem'. Consider to delete it or rename it.

'CompanySearchItem' is defined here:


(lint/suspicious/noRedeclare)